src/components/text-lockups/text-lockups.module.scss

```diff
@@ -0,0 +1,22 @@
+@import 'src/assets/stylesheets/base_styles/typography';
+@import 'src/assets/stylesheets/base_styles/colors';
+@import 'src/assets/stylesheets/utilities/functions';
+@import 'src/assets/stylesheets/utilities/mixins';
+
+.type1 {
+  $subheader-color: $neutral-4;
+
+  .header {
```

Comment:
what's the benefit of nesting here? I don't think we need to do this in general, just maybe for pseudo selectors